test/k8sT/assertionHelpers.go

```diff
@@ -120,6 +120,11 @@ func DeployCiliumOptionsAndDNS(vm *helpers.Kubectl, ciliumFilename string, optio
 		By(fmt.Sprintf("%s (hint: %s)",
 			"Assuming that microk8s already has DNS deployed...",
 			"Use 'microk8s.enable dns' to create deployment"))
+	case helpers.CIIntegrationGKE:
+		By("Restarting all kube-system pods")
+		if res := vm.DeleteResource("pod", fmt.Sprintf("-n %s --all", helpers.KubeSystemNamespace)); !res.WasSuccessful() {
+			log.Warningf("Unable to delete DNS pods: %s", res.OutputPrettyPrint())
```

Comment:
I think this msg should not be DNS-specific.

Reply:
yup!